books/free-programming-books-ko.md

```diff
@@ -237,8 +239,15 @@
 
 ### Rust
 
+* [러스트 코딩인사이트](https://coding-insight.com/docs/rust/start/rust)
+* [러스트 투어](https://tourofrust.com/00_ko.html)
+* [러스트 프로그래밍 공식가이드](https://doc.rust-kr.org/title-page.html)
 * [러스트 프로그래밍 언어](https://rinthel.github.io/rust-lang-book-ko/) - 스티브 클라브닉, 캐롤 니콜스 (HTML) *(:construction: in process)*
-* [Rust by Example](https://hanbum.gitbooks.io/rustbyexample/content/) *(:construction: in process)*
+* [마이크로소프트 러스트 강의](https://learn.microsoft.com/ko-kr/training/paths/rust-first-steps): 마이크로소프트에서 제공하는 러스트 강의
+* [예시로 배우는 러스트](https://hanbum.gitbooks.io/rustbyexample/content/) - https://github.com/rust-lang/rust-by-example
```

Comment:
Don't invent titles! Use the title given by the resource. In this case, the english title.